### PR TITLE
Add test to see if command exists with POSIX 'command'

### PR DESCRIPTION
**Pull request by exapsy**
Continuation from unverified/unsigned commits on #690

### CODE REVIEW

File: ci/e2ereport.sh [hunk 1]
```diff
@@ -9,7 +9,13 @@ FORCE_COLOR=1 DEBUG=1 go run ./e2etests/report/main.go "$@";
 
 if [ -z "${NO_OPEN:-}" ]; then
   if [ -s "$REPORT_OUTPUT" ]; then
-    open "$REPORT_OUTPUT"
+    if [ -x "$(command -v open)" ]; then
```
Comment (nhooyr) [inline ci/e2ereport.sh:12]:
```suggestion
    if command -v open >/dev/null; then
```

**Review by nhooyr — CHANGES_REQUESTED** [review]

File: ci/e2ereport.sh [hunk 2]
```diff
@@ -9,7 +9,13 @@ FORCE_COLOR=1 DEBUG=1 go run ./e2etests/report/main.go "$@";
 
 if [ -z "${NO_OPEN:-}" ]; then
   if [ -s "$REPORT_OUTPUT" ]; then
-    open "$REPORT_OUTPUT"
+    if command -v open >/dev/null; then
+      open $REPORT_OUTPUT
+    elif [ -x "$(command -v xdg-open)" ]; then
```
Comment (nhooyr) [inline ci/e2ereport.sh:14]: this one too please!

**Review by nhooyr — CHANGES_REQUESTED** [review]

File: ci/e2ereport.sh (hunk 2 above)
Comment (exapsy) [reply ci/e2ereport.sh:14]: Done!

File: ci/e2ereport.sh [hunk 3]
```diff
@@ -9,7 +9,13 @@ FORCE_COLOR=1 DEBUG=1 go run ./e2etests/report/main.go "$@";
 
 if [ -z "${NO_OPEN:-}" ]; then
   if [ -s "$REPORT_OUTPUT" ]; then
-    open "$REPORT_OUTPUT"
+    if command -v open >/dev/null; then
+      open $REPORT_OUTPUT
+    elif [ -x "$(command -v xdg-open >/dev/null)" ]; then
```
Comment (nhooyr) [inline ci/e2ereport.sh:14]:
```suggestion
    elif command -v xdg-open >/dev/null; then
```

**Review by nhooyr — CHANGES_REQUESTED** [review]

File: ci/e2ereport.sh [hunk 4]
```diff
@@ -9,7 +9,13 @@ FORCE_COLOR=1 DEBUG=1 go run ./e2etests/report/main.go "$@";
 
 if [ -z "${NO_OPEN:-}" ]; then
   if [ -s "$REPORT_OUTPUT" ]; then
-    open "$REPORT_OUTPUT"
+    if command -v open >/dev/null; then
+      open $REPORT_OUTPUT
```
Comment (nhooyr) [inline ci/e2ereport.sh:13]:
```suggestion
      open "$REPORT_OUTPUT"
```

File: ci/e2ereport.sh [hunk 5]
```diff
@@ -9,7 +9,13 @@ FORCE_COLOR=1 DEBUG=1 go run ./e2etests/report/main.go "$@";
...
+    elif command -v xdg-open >/dev/null; then
+      xdg-open $REPORT_OUTPUT
```
Comment (nhooyr) [inline ci/e2ereport.sh:15]:
```suggestion
      xdg-open "$REPORT_OUTPUT"
```

**Review by nhooyr — CHANGES_REQUESTED** [review]
apologise for all the little change requests haha

**Comment by nhooyr** [comment]
Thank you @exapsy 🚀

**Comment by alixander** [comment]
looks like you still have to approve @nhooyr

**Review by nhooyr — APPROVED** [review]
✅

**Pull request merged**